config/locales/en.yml

```diff
@@ -27,6 +27,8 @@ en:
       timestamp: '%A %-d %b %H:%M'  # Thursday 3 Jan 14:45
       snapshot: '%A %-d %B %H:%M'
       now: '%A %-d %B'
+      datestamp: '%-d %B %Y %H:%M%P' # 25 September 2024 15:29pm
+      dob: '%-d %B %Y' # 1 January 1990
```

Comment:
Double check both of these with Chris as I think the date formats need to be the same as MAAT.